src/docker.rs

```diff
@@ -248,8 +250,51 @@ pub fn run(
         }
     }
 
+    let image = if let Some(dockerfile) = config.dockerfile(target)? {
+        let mut docker_build = docker_command("build")?;
```

Comment:
Should maybe add a flag to allow specifying more flags for this command

Reply:
thinking like `DOCKER_OPTS`, I think a `DOCKER_BUILD_OPTS` is the best way to do it

Reply:
Ah ok, that's a good suggestion. Maybe also consider then #770, since we handle more than just docker now? If we're introducing new environment variables, we don't need to worry about backwards compatibility.